packages/bundle-source/exported.js

```diff
@@ -1 +1,2 @@
-import './src/types.js';
+/* eslint-disable import/export */
```

Comment:
we should try to avoid lint suppressions. How do we make the rule not complain about this case? Maybe disable it for `exported.js`.

Reply:
I see `exports.js` has all the types via `exports.d.ts`. is this file just here for backwards compatibility? Please leave a comment explanation of why they export the same.

Reply:
Yes, just for backwards compat.

Reply:
I don't see how to disable it for `exported.js` unless we turn it off entirely.  Unfortunately, it's a rule that encompasses a buch of things we probably *do* want to enforce, so I think having a suppression is much clearer.  Comment added.